### ISSUE TITLE
RAD-Y1 - clarify Resources in the Bundle

### ISSUE THREAD

**Issue by lynnfel** [IHE/RAD.IMR#8]
**Section Number** *Identify the most specific section number the issue occurs (e.g. 4.1.2)*

**Issue** 
https://build.fhir.org/ig/IHE/RAD.IMR/RAD-Y1.html#24y1412-message-semantics says: 
...shall...... FHIR Bundle Resource containing the following:
    one DiagnosticReport Resource
    one or more ServiceRequest Resources
    one Patient Resource
    one or more Organization Resources
    one or more Practitioner Resources
    one or more Observation Resources
    one or more ImagingStudy Resources

BUT
https://build.fhir.org/ig/IHE/RAD.IMR/RAD-Y1.html#24y14122-bundle-resources says:

The [IMR Bundle](https://build.fhir.org/ig/IHE/RAD.IMR/StructureDefinition-imr-bundle.html):
    shall be a [Transaction Bundle](http://hl7.org/fhir/http.html#transaction)
    shall create one [IMR DiagnosticReport]
    may create/update/read one or more [IMR ServiceRequest]
    may create/update/read one [Patient]
    may create/read one or more [Organization]
    may create/read one or more [Practitioner]
    may create/read one or more [IMR Observation]
    may create/read one or more [IMR ImagingStudy]

**Proposed Change**   I am confused by the discrepancy between the two lists, e.g. the first list contains a shall with all resources having "one" or "one or more".   Whereas, the second list contains "may" for many of the resources, which implies that the bundle may omit them??

I think there is something simple I am missing or not understanding  (Note when I refactored RAD-Y1, I removed the first list as redundant).

- Low: Typo or other minor classification that an editor can manage. Requires no group discussion.

**Comment by kinsonho** [comment]
I actually copied the model from MHD. https://profiles.ihe.net/ITI/MHD/ITI-65.html#2365412-message-semantics. The only change I made is that the top list is in bullet form.

Technically the top list describes what IMR Bundle has, while the second list describes what action the Bundle allows for each resource. So the 'may' refers to the 'create', 'update' and/or 'read' action, not the actual resource.
